rust/Cargo.toml

```diff
@@ -17,8 +17,14 @@ uuid = "1"
 crossbeam = { version = "0.8.2", features = ["crossbeam-channel"] }
 anyhow = "1.0.75"
 log = "0.4.20"
+env_logger = "0.11"
 
-anni-playback = { git = "https://github.com/ProjectAnni/anni" }
+anni-playback = { git = "https://github.com/ProjectAnni/anni", rev = "9207e60" }
+anni-player = { git = "https://github.com/snylonue/anni-player", rev = "466a434" }
 cpal = { version = "0.15.3", features = ["oboe-shared-stdcxx"] }
 rand = "0.8.5"
 material-colors = { version = "0.3.3", features = ["image"] }
+
+[patch.crates-io]
+cpal = { git = "https://github.com/sidit77/cpal.git", branch = "master" }
```

Comment:
What's the difference :thinking:

Reply:
see RustAudio/cpal#754

Reply:
I may choose to fork the repo and cherry-pick those changes to our repo so that the build system would not fail if the author deleted the related code. But this can be postponed. I may do it later.